### PR TITLE
Issue 254 Upgrade Ruby version

### PR DESCRIPTION
**Pull request by tungleduyxyz**
Related issue: https://github.com/killbill/killbill-cloud/issues/254

### CODE REVIEW

**Comment by tungleduyxyz** [comment]
The PR is ready to review @reshmabidikar

**Review by reshmabidikar — APPROVED** [review]
Looks good to me.

File: kpm/Gemfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
+gem 'sorted_set', '~> 1.0', '>= 1.0.3'
```
Comment (pierre) [inline kpm/Gemfile:4]: To remove

**Review by pierre — APPROVED** [review]
Good to merge once the new gem is removed in the Gemfile.

**Pull request merged**